src/status_im/subs/wallet/wallet.cljs

```diff
@@ -595,6 +595,11 @@
  :<- [:wallet/search-address]
  :-> :valid-ens-or-address?)
 
+(rf/reg-sub
+ :wallet/searching-address?
+ :<- [:wallet/search-address]
+ :-> :loading?)
+
```

Comment:
Added a `loading?` key to the `search-address` to know if we are ready to show errors if any or not